sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2022-06-02)
 
 ### Features Added
 
+- Support Atlas 2.2 APIs
+
 ### Breaking Changes
```

Comment:
CI failed on CHANGELOG part, please remove 'Breaking Changes' and 'Other Changes' sections as there is no content in these sections.